pages/who-we-are-hiring.md

```diff
@@ -18,6 +18,12 @@ We're actively hiring for the following positions:
 
 {% include openpositions.html %}
 
+### What's the difference bewteen a Front End Designer and Front End Web Developer?”
+
+We expect our Front End Designers to have design as well as HTML, CSS and some JavaScript experience. They should feel comfortable brainstorming and developing design options to meet user needs; developing wireframes or HTML/CSS prototypes to test their design ideas; and quickly iterating on feedback. They should also feel comfortable working as a front end developer on projects, implementing accessible HTML5 and CSS3. JavaScript experience is a plus, but they are not required to be experts.
```

Comment:
+We expect our Front End Designers to have design as well
==

Perhaps clarify "Front End Designer to have experience working with visual design and with visual designers"

==

What I'm trying to get at is that you don't have to be a Phototoshop, Sketch, or Illustrator master, however, you should know how to stumble through the software and "do" visual design basics. We're suggesting that you have a strong eye for design, even if you're not the type of person who can sit down and create custom illustrations.

If we could refine with that in mind, I think we're on the right track and help eliminate some further confusion.